Index: Probe.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import os\r\n\r\n\r\ndef read_file(path, file):\r\n    for i_elem in os.listdir(path):\r\n        if i_elem == file:\r\n\r\n\r\n    if os.path.exists(os.path.join(path, file)):\r\n        print(os.path.join(path, file))\r\n        file_read = open(os.path.join(path, file), 'r')\r\n        data = file_read.read()\r\n        print(data)\r\n        return file_read.close()\r\n        # return print(data)\r\n    for i_list in os.listdir(path):\r\n        if os.path.isdir(os.path.join(path, i_list)):\r\n            sub_path = os.path.join(path, i_list)\r\n            read_file(sub_path, file)\r\n\r\n\r\npath_search = '/home/dot/Yandex/Obsidian'\r\nfile_search = 'Template.md'\r\n\r\nread_file(path_search, file_search)\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Probe.py b/Probe.py
--- a/Probe.py	(revision b5cf60227a20c9949cc85666ff4c80b9790de576)
+++ b/Probe.py	(date 1652179488889)
@@ -4,6 +4,7 @@
 def read_file(path, file):
     for i_elem in os.listdir(path):
         if i_elem == file:
+            print(i_elem)
 
 
     if os.path.exists(os.path.join(path, file)):
